src/satosa/backends/base.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, auth_callback_func, internal_attributes, base_url, name):
         self.auth_callback_func = auth_callback_func
         self.internal_attributes = internal_attributes
         self.converter = AttributeMapper(internal_attributes)
-        self.base_url = base_url
+        self.base_url = base_url.rstrip("/") if base_url else ""
```

Comment:
The `base_url` is the `BASE`  entry in the proxy config, which is a full url not only the path. I think we don't need here the usual path canonization.